### DISCUSSION TITLE
Password Expiration Feature

### DISCUSSION THREAD

**Discussion by S1L3NTANG3L** [dani-garcia/vaultwarden#4239] [Ideas]
Hi, I have run into a problem where I have a lot of passwords that need to be updated every six months I was wondering if this could be added as a notification or icon next to the item when the password is close to the expiration set?

**Comment by BlackDex** [comment]
That would be a Bitwarden wide feature and something clients need to support.
Since Vaultwarden is only the server backend, and does not maintain any client, this question is best suited for the Bitwarden forums.

There might already be a similar feature request on there forum.
https://community.bitwarden.com/c/feature-requests/5
